docs/FAQs.md

```diff
@@ -106,7 +106,7 @@ First, I'd suggest reading [TypeScript Handbook > Modules - Introduction](https:
 
 Then:
 
-1. In `tsup.config.ts`, change the [tsup `format` option](https://tsup.egoist.dev/#bundle-formats) from `["esm"]` to `["cjs", "esm"]`
+1. In `tsdown.config.ts`, change the [tsdown `format` option](https://tsdown.dev/options/output-format) from `["esm"]` to `["cjs", "esm"]`
 2. Add a [`package.json` `"exports"` entry](https://nodejs.org/api/packages.html#subpath-exports) like:
```

Comment:
FYI - tsdown can [auto-generate the exports](https://tsdown.dev/options/package-exports#auto-generating-package-exports)

Reply:
Ooh very interesting. For the sake of keeping this change minimal I'll keep it off for now. But once it's stable, I could definitely see it being a contender for CTA.